tests/integration_tests/resources/dsl/sleep_workflows.yaml

```diff
@@ -15,6 +15,7 @@ node_templates:
 
 workflows:
     sleep: mock_workflows.mock_workflows.workflows.sleep
+    simple_sleep: mock_workflows.mock_workflows.workflows.simple_sleep
```

Comment:
Why do we now have sleep and simple_sleep? Is sleep not simple already?

Reply:
it's not, the main sleep also does some runtime_props stuff
anyway, we've always had simple_sleep, this just exposes it

Reply:
@geokala good question, this is because `sleep` update the `states` which is something I do not want to have in the testa and simple_sleep is just sleep without anything